Bug/cscli: SIGSEGV when running cscli with missing configuration

**Describe the bug**
When trying to run cscli with missing mandatory sections in `/etc/crowdsec/config.yaml`, it segfaults.

**To Reproduce**
Steps to reproduce the behavior:
With the following configuration (note the missing `cscli` section):
```yaml
common:
  daemonize: false
  log_media: stdout
  working_dir: .
config_paths:
  config_dir: /etc/crowdsec/
  data_dir: /var/lib/crowdsec/data/
  simulation_path: /etc/crowdsec/simulation.yaml
  hub_dir: /etc/crowdsec/hub/
  index_path: /etc/crowdsec/hub/.index.json
db_config:
  log_level: info
  type: sqlite
  db_path: /tmp/crowdsec.db
api:
  server:
    log_level: info
    listen_uri: 127.0.0.1:8080
    profiles_path: /etc/crowdsec/profiles.yaml
```

```shell
~/git/crowdsec (master ✘)✖✹✭ ᐅ ./cmd/crowdsec-cli/cscli hub update
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xde53ea]

goroutine 1 [running]:
main.initConfig()
	/home/sebastienblot/git/crowdsec/cmd/crowdsec-cli/main.go:50 +0x13a
github.com/spf13/cobra.(*Command).preRun(0xc0003c1080)
	/home/sebastienblot/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:880 +0x49
github.com/spf13/cobra.(*Command).execute(0xc0003c1080, 0x1a5cb18, 0x0, 0x0, 0xc0003c1080, 0x1a5cb18)
	/home/sebastienblot/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:816 +0x145
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000e3600, 0xc000515f18, 0x1, 0x1)
	/home/sebastienblot/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/home/sebastienblot/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
	/home/sebastienblot/git/crowdsec/cmd/crowdsec-cli/main.go:138 +0xb31
```

It also happens if `common`, `config_paths` or `api` are missing.

**Expected behavior**
An error message telling me the configuration is missing mandatory sections.

**Technical Information (please complete the following information):**
 - Version: 1.0.6

--- a/cmd/crowdsec-cli/main.go
+++ b/cmd/crowdsec-cli/main.go
@@ -47,6 +47,10 @@ func initConfig() {
 	if err := csConfig.LoadConfigurationFile(ConfigFilePath); err != nil {
 		log.Fatalf(err.Error())
 	}
+	if csConfig.Cscli == nil {
+		log.Fatalf("missing 'cscli' configuration in '%s', exiting", ConfigFilePath)
+	}
+
 	if cwhub.HubBranch == "" && csConfig.Cscli.HubBranch != "" {
 		cwhub.HubBranch = csConfig.Cscli.HubBranch
 	}

